static/docs/commands-reference/move.md

```diff
@@ -59,17 +62,20 @@ name:
 
 ```dvc
 $ dvc move data.csv other.csv
-$ cat data.csv.dvc
+$ cat other.csv.dvc
 ```
 
-And here is the updated content of the `data.csv.dvc`:
+And here is the updated content of the `other.csv.dvc`:
 
 ```yaml
-md5: 3d1a3e5a5b662490e198d6a6ae84984b
+md5: df33f383592e7a399e106ba1cb487a8c
+wdir: .
```

Comment:
let's skip some default values for the sake of simplicity, no need to include wdir, cahce, persist everywhere

Reply:
`metrics` need to be removed too?
@shcheklein

Reply:
discussed in private, yep, let's remove default metric value where it does not make much sense. In general, let's add/remove fields only if it's clear that it's not a default value, or if it can make easier to understand what's happening